run_dir/static/js/project_samples.js

```diff
@@ -1179,7 +1179,11 @@ function load_charon_summary(){
           $('#charon-status-failed').text(data['failed']);
           $('#charon-status-runn').text(data['runn']);
           $('#charon-status-hge').text(data['hge']);
-          $('#charon-status-gdp').text(data['gdp'].toString().replace(/\'/," "));
+          var del_projs = '';
+          data['gdp'].forEach(function(del_proj){
+              del_projs += '<span class="badge bg-secondary">'+del_proj+'</span>'+' ';
```

Comment:
Use padding instead
```suggestion
              del_projs += '<span class="badge bg-secondary p-1">'+del_proj+'</span>';
```
maybe?